volcli/volcli.go

```diff
@@ -133,7 +134,11 @@ func tenantUpload(ctx *cli.Context) (bool, error) {
 		return false, err
 	}
 
-	tenant := &config.TenantConfig{}
+	tenant := &config.TenantConfig{
```

Comment:
there's a lot of repetition about this. Let's instead make  amethod, `config.NewTenantConfig(driver string)` that performs this construction. This can also be used in the tests where you do this.

Does this work for you?